### DISCUSSION TITLE
D365 store commerce (10.0.33) : customer display  - Not able to Zoom-in/out

### DISCUSSION THREAD

**Discussion by Shubham-SauravGMG** [microsoft/Dynamics365Commerce.Solutions#188] [Q&A]
Hi,

After performing the store commerce upgrade on 20/7/2023 we are facing issue with Customer Display, the dual screen is not showing the total amount.
I have attached screenshot of the issue from "RSH and Vans Setia City store" for reference.

![D365_Vans_02_customer display (1)](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/144985335/624eb8a0-0a18-442a-81f8-31385bf8dd5d)
![D365_Vans_01_customer display (1)](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/144985335/202611b1-5346-4d05-9471-43fd99ba2c2a)
![D365_RSHsetia_02_customer display (1)](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/144985335/dabe7cdf-972e-4126-addc-98d0b9de8899)

Please provide us an immediate fix for the issue as the issue is directly impacting customer experience in more than 250 stores and we have raised this issue in July with Microsoft.

Regards,
Shubham
Email - c_shubham.s@gmg.com

**Comment by ViktorVu602** [comment]
Hi @madyke, as this is issue has been pending for a long time, may I ask if you or someone on your team can help to look into this issue?
I will attach customer's solution for you via email.

**Reply by madyke** [reply]
Hi @ViktorVu602 - this looks like a potential bug with the MS code rather than an extension design question or issue. Can you have support file a bug for this through the normal channel and assign it to our team to investigate?

**Reply by ViktorVu602** [reply]
Hi @madyke, thanks for the response. I will discuss with Guanghui on reproducing this issue.
